thinc/config.py

```diff
@@ -128,8 +128,12 @@ class Config(dict):
     under the hood.
     """
 
+    is_interpolated: bool
+
     def __init__(
-        self, data: Optional[Union[Dict[str, Any], "ConfigParser", "Config"]] = None
+        self,
+        data: Optional[Union[Dict[str, Any], "ConfigParser", "Config"]] = None,
+        is_interpolated: Optional[bool] = None,
```

Comment:
We probably want this keyword only? Maybe there's a reason to allow positional, but I'm trying to remember to do the KWO args, so I'm trying to remember to suggest it in reviews.

```suggestion
        *,
        is_interpolated: Optional[bool] = None,
```

Reply:
Would it be better to make the argument name `interpolate`? I'm not sure.

To me `is_interpolated` leans a little more towards describing the state of the data being passed in. I'd normally prefer a verb to indicate whether to perform the action.

I would definitely keep the `is_interpolated` name for the boolean, though.

Reply:
> To me is_interpolated leans a little more towards describing the state of the data being passed in.

Yes, that's the intention. The `is_interpolated` argument here is purely to note the state of the data being passed in, if the `Config` is created from a dict. So it's mostly a shortcut for setting `config.is_interpolated` on a separate line.